### DISCUSSION TITLE
不在YouTube中翻译视频字幕

### DISCUSSION THREAD

**Discussion by eAtlantis000** [immersive-translate/immersive-translate#464] [Help] [answered]
### 概述 | Summary

个人是码盲，希望通过能实现在YouTube中不对视频字幕进行翻译但是可翻译页面文字内容的设置，希望得到指导，谢谢回答的好心人

**Comment by theowenyoung** [comment, marked as answer]
设置为永不翻译，然后需要的时候再手动点击翻译

eAtlantis000 ***@***.***> 于 2023年5月13日周六 下午10:53写道：
